test/thermodynamics/maxwell_boltzmann_distributions/speed_distribution_test.py

```diff
@@ -0,0 +1,55 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (
+    assert_equal,
+    errors,
+    units,
+    Quantity,
+)
+from symplyphysics.core.symbols.celsius import Celsius, to_kelvin_quantity
+from symplyphysics.laws.thermodynamics.maxwell_boltzmann_distributions import (
+    speed_distribution as distribution_law,
+)
+
+# Description
+## The value of the speed distribution function for an ensemble of Argon particles (particle mass m = 39.948 u)
+## at equilibrium temperature T = 10°C at speed v = 10 m/s is 5.57e-6 s/m.
```

Comment:
`seconds / meter` is valid unit of measurement, but it reflects amount of particles per unit speed, so I think it is better to use `1 / (meter / second)`

Reply:
Sure